solarforecastarbiter/io/api.py

```diff
@@ -1321,3 +1328,44 @@ def _forecast_get_by_type(self, forecast_type):
             return self.get_probabilistic_forecast_constant_value
         else:
             raise ValueError('Invalid forecast type.')
+
+    @ensure_timestamps('start', 'end')
+    def _get_values(self, api_path, start, end, parse_fn, params={}):
+        """Breaks up a get requests for values into multiple requests limited
+        by the GET_VALUES_LIMIT value.
```

Comment:
I would also like to be able to set this on a per get_*_basis so maybe add a kwarg that defaults to it